clearly-defined/src/main/kotlin/ClearlyDefinedService.kt

```diff
@@ -41,8 +41,6 @@ import retrofit2.http.PATCH
 import retrofit2.http.POST
 import retrofit2.http.Path
 
-const val HARVEST_CREATED = "Created"
-
```

Comment:
Please keep this one. It will be used again once the Harvest API is used to trigger scans.

Reply:
Removed the commit.